Vesna rosh patch 1

VesnaRosh opened the pull request.

VesnaRosh committed:
Create hello.txt

VesnaRosh committed:
Add files via upload

VesnaRosh committed:
Create hello.txt

VesnaRosh committed:
Add files via upload

VesnaRosh committed:
Create hello.txt

VesnaRosh committed:
Add files via upload

VesnaRosh committed:
Create hello.txt

VesnaRosh committed:
Add files via upload

VesnaRosh committed:
Create hello.txt

Metodija-Mitovski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/about-us.html:
```diff
@@ -0,0 +1,61 @@
+    <title>About us</title>
+</head>
+<body>
+    <div>  
```
mozebi ovoj div ne ti e potreben

VesnaRosh replied:
Vo pravo si, i za boldot, merci :)

Metodija-Mitovski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/unordered-ordered-list.html:
```diff
@@ -0,0 +1,73 @@
+    <table>
+        <tr>
+            <td>
+                <H3><b>Unordered list&nbsp;</b></H3>
```
ne ti e potreben bold na heading

Metodija-Mitovski reviewed (commented):
dobro ureden kod :)

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/categories.html:
```diff
@@ -0,0 +1,124 @@
+        <ul type="none">
+            <li>1) Monday
+                <ul type="none">
+                    <li>a) 10:00 a.m.
```
mozhesh so `type="a"` da go sredish.

VesnaRosh replied:
So type="a", se printa "a." (so tocka) . So ogled na sto na slikata bese a), se obidov da go izvedam toa "a)" (so zagrada) bez da koristam CSS.  Zatoa koristev type="none", za da nema bullets. Sega koga napomenavte deka ne e bitno dali e tocka ili zagrada, kje go korigiram posledovatelno tamu kaj sto sum go upotrebila :)

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/categories.html:
```diff
@@ -0,0 +1,124 @@
+        </ul>
+      <hr/>
+        <span>Weekly plan:</span>
+        <ul type="none">
```
zosto type="none", sto oznacuva toa?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/categories.html:
```diff
@@ -0,0 +1,124 @@
+      <hr/>
+        <span>Weekly plan:</span>
+        <ul type="none">
+            <li>1) Monday
```
ne treba da bidi 1). , samata lista dodeluva oznaka.

emillija replied:
deka so zagrada go pisuvase na slikata za domasna, jas za ova te prasav kade da najdam ) i mi rece deka moze sami da si stavime )

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/features.html:
```diff
@@ -0,0 +1,49 @@
+    <ul>
+        <li>You can meet people</li>
+        <li>You can have access
+            <ul type="">
```
tuka nedostasuva vrednost na type ili e visok ova?

VesnaRosh replied:
Propust e, ne sum go izbrisala.

VesnaRosh committed:
Update features.html

VesnaRosh committed:
Update about-us.html

VesnaRosh committed:
Update categories.html

VesnaRosh committed:
Update about-us.html

VesnaRosh committed:
Update gallery.html

VesnaRosh committed:
Update rosh-photo-home.html

VesnaRosh committed:
Delete hello.txt

VesnaRosh committed:
Delete hello.txt

VesnaRosh committed:
Update landscape.html

VesnaRosh committed:
Update wedding.html

VesnaRosh committed:
Update landscape.html

VesnaRosh committed:
Delete hello.txt

VesnaRosh committed:
Update multi-layered.html

VesnaRosh committed:
Update unordered-ordered-list.html

VesnaRosh committed:
Add files via upload

VesnaRosh committed:
Update google-docs.html

@dpeshevski  Vaka moze? Za da ispadne nabrojuvanjeto kako na slikata sto e vo outcomes i vo domasnoto? Ili ne e ok?

VesnaRosh committed:
Create hello.txt

VesnaRosh committed:
Add files via upload

meto2208 commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/gallery.html:
```diff
@@ -0,0 +1,53 @@
+    </header>
+    <main>
+        <h1>Our work</h1>
+        <caption>
```
Весна caption елемент се користи како наслов на табела и секогаш се користи во table елемент.
Поздрав.

VesnaRosh replied:
Vo pravo si, treba da e <figure>.

meto2208 commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/multi-layered.html:
```diff
@@ -0,0 +1,46 @@
+    <title>Document</title>
+</head>
+<body>
+    Preceding text
```
Можеш да го спакуваш во heading или друг елемент.
Поздрав

meto2208 replied:
Се друго во homework-01 е супер и многу читливо.
Поздрав

VesnaRosh committed:
Update gallery.html

VesnaRosh committed:
Update multi-layered.html

VesnaRosh committed:
Delete students/Vesna_Rosh/homeworks/homework-02 directory

VesnaRosh committed:
Delete hello.txt

VesnaRosh committed:
Delete hello.txt

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/gallery.html:
```diff
@@ -0,0 +1,66 @@
+            <a href="categories.html" target="_blank">Categories|</a>
+        </nav>
+    </header>
+  </div width="100%">
```
zosto `<div>`?

VesnaRosh replied:
Gledav sto moze so div da se napravi i ne sum doizbrisala.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/gallery.html:
```diff
@@ -0,0 +1,66 @@
+  </div width="100%">
+    <main>
+        <h1>Our work</h1>
+        <figure>
```
sto oznacuva ovoj element, zosto se koristi?

VesnaRosh replied:
Za sliki, dijagrami, ilustracii, zaedno so figcaption za komentar na slika. Pogresno li go upotrebuvam i sfakjam?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/gallery.html:
```diff
@@ -0,0 +1,66 @@
+    <main>
+        <h1>Our work</h1>
+        <figure>
+            <figcaption>
```
zosto se koristi ovoj element?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/google-docs.html:
```diff
@@ -0,0 +1,53 @@
+    <title>Document</title>
+</head>
+<body>
+    <p><u><b>Google Docs - areas to learn</b></u></p>
```
zosto `<u>` i `<b>`? zarem ne mozhe so heading tag da se napravi?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/movies.html:
```diff
@@ -0,0 +1,60 @@
+            <li>House</li>
+        </ol>
+    <h2>Definitions</h2> 
+        <dl>
```
zosto se koristi `<dl>`?

VesnaRosh replied:
Za opisna lista,  pr lista od termini so definicii. DT, definition term, e za terminot, DD, definition description za objasnuvanjeto, i se indentiraat soodvetno.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/movies.html:
```diff
@@ -0,0 +1,60 @@
+        <dl>
+            <dt>daft
+                <dd>sensless, stupid, or foolish
+            <dt>prerequisite
```
zosto se koristi `<dt>`?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/ordered-list-types.html:
```diff
@@ -0,0 +1,43 @@
+</head>
+<body>
+    <h1>Oredered List Types</h1>
+    Default 
```
**Default** kako text ne staven vo element.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/ordered-list-types.html:
```diff
@@ -0,0 +1,43 @@
+        <li>Bannana</li>
+        <li>Apple</li>
+    </ol>
+    Lower Alphabets
```
Isto i ova.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/ordered-list-types.html:
```diff
@@ -0,0 +1,43 @@
+        <li>Electrisity</li>
+        <li>Transportation</li>
+    </ol>
+    Lower Roman
```
tuka.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/ordered-list-types.html:
```diff
@@ -0,0 +1,43 @@
+        <li>Green</li>
+        <li>Yelllow</li>
+    </ol>
+    Upper Alphabets
```
tuka.

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/unordered-ordered-list.html:
```diff
@@ -0,0 +1,73 @@
+        <tr>
+            <td>
+                <h3>Unordered list&nbsp;</h3>
+                <b>Disc</b>
```
zosto `<b>`?

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/unordered-ordered-list.html:
```diff
@@ -0,0 +1,73 @@
+</head>
+<body>
+  <div>
+    <table>
```
zosto `<table>`??

dpeshevski commented on students/Vesna_Rosh/homeworks/homework-01/the-other-solutions/unordered-ordered-list.html:
```diff
@@ -0,0 +1,73 @@
+            </td>
+            <br/>
+            <td>
+                <h3><b>Ordered list&nbsp;</b></h3>
```
ne ti treba `<b>` dokolku koristis heading tag el.

Nikolina5 commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/about-us.html:
```diff
@@ -0,0 +1,60 @@
+            <br/>
+        </p>
+        <hr/>
+        <span>Related posts:</span>
```
Ova bi trebalo mesto span da e vo paragraf. Span e za del od recenica makar i kratka koja se naogja vo paragraf. :)

VesnaRosh replied:
Jas smetam deka se koristi za stiliziranje zbor ili fraza, be razlika dali e vo p tag. Nitu Daniel na cas ne go stavi span vo paragraf. Ako mu stavam stil, primer da mi bidat bukvite crveni, kje ti ima logika? Sto mislis?

Nikolina5 commented on students/Vesna_Rosh/homeworks/homework-01/my-pages/about-us.html:
```diff
@@ -0,0 +1,60 @@
+    </main>
+    <hr/>
+    <footer>
+        <span>Contact:</span>
```
I za kontakt e istoto <3 👍

Nikolina5 reviewed (approved):
Lesno citliv i ureden kod. Potrebni se mali izmeni, no veke izgleda profesionalno !! Bravo :)

emillija reviewed (approved):
Super odlicno

VesnaRosh committed:
Update gallery.html

VesnaRosh committed:
Update google-docs.html

VesnaRosh committed:
Update ordered-list-types.html

VesnaRosh committed:
Update unordered-ordered-list.html

VesnaRosh committed:
Update unordered-ordered-list.html

emillija reviewed (approved):
super ureden kod :)))))))))))
